packages/genesys-spark-components/src/components/beta/gux-selector-cards/gux-selector-card/gux-selector-card.scss

```diff
@@ -118,74 +106,146 @@ $gux-icon-radio-checked: "data:image/svg+xml,%3Csvg viewBox='0 0 16 16' xmlns='h
-      );
+        @include unchecked;
+      }
 
-      @include unchecked;
+      ::slotted(input[type='radio']:checked:not(:disabled))::before {
+        background-color: var(
+          --gse-ui-selectorCard-active-selectedIndicator-foregroundColor
+        );
+
+        @include checked;
+      }
     }
 
-    ::slotted(input[type='radio']:checked:not(:disabled))::before {
-      background-color: var(
-        --gse-ui-selectorCard-active-selectedIndicator-foregroundColor
-      );
+    .gux-screenreader {
+      @include mixins.gux-sr-only;
+    }
+  }
 
-      @include checked;
+  &.gux-descriptive {
+    .gux-content {
+      align-items: flex-start;
+      width: var(--gse-ui-selectorCard-descriptive-minWidth);
+      min-width: var(--gse-ui-selectorCard-descriptive-minWidth);
+      max-width: var(--gse-ui-selectorCard-descriptive-maxWidth);
+      min-height: var(--gse-ui-selectorCard-descriptive-minHeight);
+      max-height: var(--gse-ui-selectorCard-descriptive-maxHeight);
+      padding: var(--gse-ui-selectorCard-descriptive-padding);
+      text-align: start;
+      border-radius: var(--gse-ui-selectorCard-descriptive-borderRadius);
+
+      .gux-description {
+        margin-bottom: var(--gse-ui-selectorCard-descriptive-gap);
+        font-family: var(
+          --gse-ui-selectorCard-descriptive-description-fontFamily
+        );
+        font-size: var(--gse-ui-selectorCard-descriptive-description-fontSize);
+        font-weight: var(
+          --gse-ui-selectorCard-descriptive-description-fontWeight
+        );
+        line-height: var(
+          --gse-ui-selectorCard-descriptive-description-lineHeight
+        );
+      }
+
+      ::slotted(gux-badge) {
+        margin-top: var(-gse-ui-selectorCard-descriptive-badge-marginTop);
+      }
+
+      ::slotted(input[type='radio']) {
+        left: 0;
+        border-radius: var(--gse-ui-selectorCard-descriptive-borderRadius);
+
+        &::before {
+          inset-block-start: var(--gse-ui-selectorCard-descriptive-gap);
+          inset-inline-end: var(--gse-ui-selectorCard-descriptive-gap);
+          width: var(--gse-ui-icon-size-md);
+          height: var(--gse-ui-icon-size-md);
+        }
+      }
+
+      ::slotted(label) {
+        margin-bottom: var(--gse-ui-selectorCard-descriptive-text-gap);
+        font-family: var(--gse-ui-selectorCard-descriptive-label-fontFamily);
+        font-size: var(--gse-ui-selectorCard-descriptive-label-fontSize);
+        font-weight: var(--gse-ui-selectorCard-descriptive-label-fontWeight);
+        line-height: var(--gse-ui-selectorCard-descriptive-label-lineHeight);
+      }
+
+      ::slotted(gux-icon) {
+        width: var(--gse-ui-icon-size-lg);
+        height: var(--gse-ui-icon-size-lg);
+        margin-bottom: var(--gse-ui-selectorCard-descriptive-gap);
+      }
     }
   }
 
-  .gux-screenreader {
-    @include mixins.gux-sr-only;
+  &.gux-disabled {
+    cursor: not-allowed;
```

Comment:
I think `pointer-events:none` will be needed here as you can still select a `disabled` radio button.